src/api/migraphx.py

```diff
@@ -182,6 +189,11 @@ def program_parameters(h):
              api.params(name='const char*',
                         argument='const migraphx::argument&'),
              invoke='${program_parameters}[${name}] = ${argument}')
+    h.method('contains',
```

Comment:
I dont see why we need to expose this in the API.

Reply:
I will remove it, used it previously. Not used anymore.